### PR TITLE
Doc: add wstETH on Zircuit

### PR DESCRIPTION
**Pull request by TheDZhon**
**Context**

- https://research.lido.fi/t/wsteth-deployment-to-zircuit-and-ownership-acceptance-by-lido-dao/8602
- audit: https://github.com/lidofinance/audits/blob/main/L2/Zircuit_2024-10-02-Quantstamp-wstETH-deployment-verification.pdf
- state-mate cfg: https://github.com/lidofinance/state-mate/blob/main/configs/zircuit/mainnet.yml

### CODE REVIEW

File: docs/multisigs/emergency-brakes.md [hunk 1]
```diff
@@ -364,7 +364,7 @@ L2ERC20TokenBridge: Mode:[`0xb8161F28a5a38cE58f155D9A96bDAc0104985FAc`](https://
 
 **Contracts and Roles:**
 
-L2ERC20TokenBridge: Zircuit:[`0x9Bff79BF7226cB5C16d0Cca9c1dc60450feE560d`](https://safe.zircuit.com/settings/setup?safe=zircuit-mainnet:0x9Bff79BF7226cB5C16d0Cca9c1dc60450feE560d)
+L2ERC20TokenBridge: Zircuit:[`0x7721F53d153Ae3CF937605fF1Bbb7D51B14E7902`](https://explorer.testnet.zircuit.com/address/0x7721F53d153Ae3CF937605fF1Bbb7D51B14E7902)
```
Comment (iamnp) [inline docs/multisigs/emergency-brakes.md:367]: @TheDZhon this seems to be for the Testnet?

Comment (TheDZhon) [reply docs/multisigs/emergency-brakes.md]: yeah, good catch! :+1: fixed

**Review by arwer13 — APPROVED** [review]
The addresses match the addresses from the research forum post and the proposed changes in general seem to be correct.

File: docs/multisigs/emergency-brakes.md [hunk 2]
```diff
@@ -95,6 +95,12 @@ GateSeal [`0x1ad5cb2955940f998081c1ef5f5f00875431aa90`](https://etherscan.io/add
 
         - pause capability
 
+
+###### Zircuit Token Bridge: [`0x912C7271a6A3622dfb8B218eb46a6122aB046C79`](https://etherscan.io/address/0x912C7271a6A3622dfb8B218eb46a6122aB046C79)
```
Comment (iamnp) [inline docs/multisigs/emergency-brakes.md:99]: @TheDZhon, we do not use the (proposed) mark in the emergency-brakes.md file, correct?

Comment (TheDZhon) [reply docs/multisigs/emergency-brakes.md:99]: yeah, see no issue here tbh

**Review by grstepanov — APPROVED** [review]
lgtm

**Pull request merged**